src/map/modifier.h

```diff
@@ -412,6 +412,7 @@ enum class Mod
     UTSUSEMI_BONUS            = 900, // Extra shadows from gear
     NINJA_TOOL                = 308, // Percent chance to not use a tool.
     NIN_NUKE_BONUS            = 522, // magic attack bonus for NIN nukes
+    DAKEN                     = 909, // chance to throw a shuriken without consuming it
```

Comment:
The mod id needs to be updated after merging master. I believe the next spare is 911 but definitely double check that. Update here, status.lua, and traits.sql.

Reply:
![image](https://user-images.githubusercontent.com/3845823/40588768-b93d0db0-621d-11e8-91eb-7488e0658ebb.png)